sasmodels/TwoYukawa/testGr.py

```diff
@@ -0,0 +1,42 @@
+from numpy import mean
```

Comment:
This is a duplicate of a function in `CalTYSK.py` and this particular implementation is never called as far as I can tell. Remove it, maybe?